drivers/clock_control/clock_control_mcux_mcg.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 Vestas Wind Systems A/S
+ *
+ * Based on clock_control_mcux_sim.c, which is:
+ * Copyright (c) 2017, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <clock_control.h>
```

Comment:
This should now be `drivers/clock_control.h`

Reply:
@MaureenHelm Fixed, thanks.